Mage.Sets/src/mage/cards/g/Glaciers.java

```diff
@@ -0,0 +1,104 @@
+package mage.cards.g;
+
+import mage.abilities.Ability;
+import mage.abilities.common.BeginningOfUpkeepTriggeredAbility;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.costs.mana.ManaCostsImpl;
+import mage.abilities.effects.ContinuousEffect;
+import mage.abilities.effects.ContinuousEffectImpl;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.common.SacrificeSourceUnlessPaysEffect;
+import mage.abilities.mana.WhiteManaAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.filter.common.FilterLandPermanent;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+/**
+ *
+ * @author jmharmon
+ */
+
+public final class Glaciers extends CardImpl {
+
+    private  static final FilterLandPermanent filter = new FilterLandPermanent(SubType.MOUNTAIN, "Mountains");
```

Comment:
move the filter to the static inner class

Reply:
This should be as is. The card is the same as Covnersion, excpet for mana cost and upkeep cost.

Reply:
Did you actually test this card?  The layer 6 part of it and Conversion do not work on my IDE.  The type change layer 4 is fine.

Reply:
It's been a few day since I have, it should be working ok. I don't remember the exact results. I'll test it again later.

Reply:
It should be fine once a few adjustments are made.